Source/Core/UICommon/UICommon.cpp

```diff
@@ -66,6 +69,55 @@ void Shutdown()
   Config::Shutdown();
 }
 
+void SetLocale(std::string locale_name)
+{
+  auto set_locale = [](const std::string& locale) {
+    // setlocale sets the C locale, and global sets the C and C++ locales, so the call to setlocale
+    // would be redundant if it wasn't for not having any other good way to check whether
+    // the locale name is valid. (Constructing a std::locale object for an unsupported
+    // locale name throws std::runtime_error, and exception handling is disabled in Dolphin.)
+    if (!std::setlocale(LC_ALL, locale.c_str()))
+      return false;
+    std::locale::global(std::locale(locale));
+    return true;
+  };
+
+#ifdef _WIN32
+  constexpr char preferred_separator = '-';
```

Comment:
This should be fully uppercased (i.e. `PREFERRED_SEPARATOR`) as it's a compile-time constant, I think.